client/my-sites/checkout/composite-checkout/test/composite-checkout.js

```diff
@@ -587,7 +587,7 @@ describe( 'CompositeCheckout', () => {
 		getAllByLabelText( 'WordPress.com Personal' ).map( ( element ) =>
 			expect( element ).toHaveTextContent( 'R$144' )
 		);
-		getAllByLabelText( 'Domain Mapping: bar.com' ).map( ( element ) =>
+		getAllByLabelText( 'bar.com' ).map( ( element ) =>
```

Comment:
Does this display the product name anywhere? This change could mean that I could introduce a bug where we display `bar.com` in the page header but the mapping product doesn't correctly get added to the cart and this test would still pass.